osf/models/spam.py

```diff
@@ -221,6 +224,13 @@ def do_check_spam(self, author, author_email, content, request_headers, update=T
         if self.is_spammy:
             return True
 
+        enqueue_task(
```

Comment:
What we learned from the `sync_doi_metadata` command is that `enqueue_task` is apparently annoying and doesn't always do what we want. This is _probably_ fine, since this will almost always run in a request context, but we should just default to <task>.apply_async so that we don't have a bunch of different ways that we're running things asynchronously.

For an example, see: https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/management/commands/sync_doi_metadata.py#L36